internal/distro/rhel9/images.go

```diff
@@ -55,9 +55,13 @@ func osCustomizations(
 		osc.ExcludeDocs = *imageConfig.ExcludeDocs
 	}
 
-	if !t.bootISO {
+	if !t.bootISO || (t.name == "edge-simplified-installer" && !common.VersionLessThan(t.arch.distro.osVersion, "9.2")) {
```

Comment:
What is the actual intention of this condition?

If it is to set the user / group customization for edge-simplified-installer, then this is not the right place to do it, since the image type does not use the `os` pipeline at all. The customization must be instead set in the commit-deploy pipeline. 

Looking at https://github.com/osbuild/osbuild-composer/blob/96b42b57919440b798b2171f45792efcf1c148f1/internal/distro/rhel9/images.go#L409-L410 everything seems to be already hooked up.

Reply:
very true :+1: